CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# dbt_google_ads v1.0.1
+
+[PR #89](https://github.com/fivetran/dbt_google_ads/pull/89) includes the following updates:
+
+## Under the Hood
+- Introduced `partition_by_source_relation` macro to dynamically handle source_relation partitioning based on union configuration.
```

Comment:
Should this be called out as a bug fix?

Reply:
Since this is addressing an issue which could arise as a result of the source_relation field, yes this should be classified as a bugfix. Updated the section header.